man/coredump.conf.xml

```diff
@@ -86,6 +86,18 @@
         <listitem><para>The maximum size in bytes of a core
         which will be processed. Core dumps exceeding this size
         may be stored, but the backtrace will not be generated.
+        Like other sizes in this same config file, SI suffixes
+        are allowed (B, K, M, G, T, P, and E.) For example, 
+        1G may be used in place of 1073741824 bytes. Decimals
+        are allowed.
+
+        A series of SI numbers may be followed by a number in 
+        bytes with the suffix "B" to allow complete precision.
+        The numbers in the series will be added but they must
+        appear in decreasing SI size order.
+
+        For example "3.5G 4B" = 3*1024*1024*1024 + 512*1024*1024 + 4 =
+        3758096388 bytes.
```

Comment:
if you want multiple paragraphs, you need to use `<para></para>`. But quite frankly, please keep this short, don't add excerpt about SI

Reply:
In the name of conciseness, this whole example has been removed.